Backend/Backend/Controllers/Requests/PodcastRequest.cs

```diff
@@ -34,5 +35,37 @@ public class GetPodcastRequest
         public Files? coverImage { get; set; }
 
 
+
+
     }
+
+    [BindProperties]
+    public class GetPodcastResponse
```

Comment:
Isnt this the same thing as before?

Reply:
Not sure which of these is the right one

Reply:
No actually I have used this to return podcasts